app/connectors/SdesConnector.scala

```diff
@@ -76,14 +76,17 @@ class SdesConnector @Inject()(httpClient: HttpClientV2,
     )
   }
 
+  private def addXHeaders(hc: HeaderCarrier, key: String): HeaderCarrier =
```

Comment:
I meant like this: https://github.com/hmrc/customs-manage-authorities-frontend/blob/main/app/config/Headers.scala

Sorry, should have made that clearer.